cassandra_backup_service.py

```diff
@@ -479,10 +485,17 @@ def upload_incremental_backups(self, host_id, data_file_directory, filepath=None
             local_path = '{0}{1}'.format(data_file_directory, filepath)
             remote_path = '{0}/{1}'.format(bucket, filepath)
             if local_path.endswith('/'):
-                cmd = ['aws', 's3', 'cp', '--storage-class', 'STANDARD_IA', '--recursive', local_path, remote_path]
+                cmd = ['aws', 's3', 'cp']
+                if self.s3_storage_class:
+                    cmd += ['--storage-class', 'STANDARD_IA']
```

Comment:
Isn't this still hard coded?

Reply:
Fixed and tested for incremental backups, thanks Jose:
<img width="1440" alt="Screen Shot 2020-11-16 at 6 14 27 PM" src="https://user-images.githubusercontent.com/8575685/99337683-9ac75880-2837-11eb-9cb2-93d7e24a8dbc.png">

```
python cbs.py incremental aws --aws-s3-bucket s3://tropical-box/red_red --aws-s3-storage-class REDUCED_REDUNDANCY
```